examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+"""
+Model-based curation tutorial
+=============================
+
+This notebook provides a step-by-step guide on how to use a machine learning classifier for
+curating spike sorted output. We'll download a toy model and use it to label our sorted data by using
+Spikeinterface. We start by importing some packages
+"""
+
+import warnings
+warnings.filterwarnings("ignore")
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import spikeinterface.core as si
+import spikeinterface.curation as sc
+import spikeinterface.widgets as sw
+
+
+# note: you can use more cores using e.g.
+# si.set_global_jobs_kwargs(n_jobs = 8)
+
+##############################################################################
+# Download a pretrained model
+# ---------------------------
+#
+# Let's download a pretrained model from `Hugging Face <https://huggingface.co/>`_ (HF). The
+# ``load_model`` function allows us to download directly from HF, or use a model in a local
+# folder. The function downloads the model and saves it in a temporary folder and returns a
+# model and some metadata about the model.
+
+model, model_info = sc.load_model(
+    repo_id = "SpikeInterface/toy_tetrode_model",
+    trusted = ['numpy.dtype']
+)
+
+
+##############################################################################
+# This model was trained on artifically generated tetrode data. The model object has a nice html
+# representation, which will appear if you're using a Jupyter notebook.
+
+model
+
+##############################################################################
+# The model object (an sklearn Pipeline) contains information about which metrics
+# were used to compute the model. We can access it from the model (or from the model_info)
+
+print(model.feature_names_in_)
+
+##############################################################################
+# Hence, to use this model we need to create a ``sorting_analyzer`` with all these metrics computed.
+# We'll do this by generating a recording and sorting, creating a sorting analyzer and computing a
+# bunch of extensions. Follow these links for more info on `recordings <https://spikeinterface.readthedocs.io/en/latest/modules/extractors.html>`_, `sortings <https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html>`_, `sorting analyzers <https://spikeinterface.readthedocs.io/en/latest/tutorials/core/plot_4_sorting_analyzer.html#sphx-glr-tutorials-core-plot-4-sorting-analyzer-py>`_
+# and `extensions <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html>`_.
+
+recording, sorting = si.generate_ground_truth_recording(num_channels=4, seed=4, num_units=10)
+sorting_analyzer = si.create_sorting_analyzer(sorting=sorting, recording=recording)
+sorting_analyzer.compute(['noise_levels','random_spikes','waveforms','templates','spike_locations','spike_amplitudes','correlograms','principal_components','quality_metrics','template_metrics'])
```

Comment:
No, the `feature_names_in` are just the individual quality and template metrics

Reply:
Note to self: if we unify the quality and template metrics it would be easy to make a funciton `compute_all_metrics(metric_names=model.feature_names_in_)`